cep-??.md

```diff
@@ -0,0 +1,246 @@
+<table>
+<tr><td> Title </td><td> The <code>MatchSpec</code> grammar </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Jaime Rodríguez-Guerra &lt;jaime.rogue@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> June 4, 2024 </td></tr>
+<tr><td> Updated </td><td> June 4, 2024 </td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda/ceps/pull/82 </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+This CEP standardizes the grammar for the `MatchSpec` query language.
+
+## Motivation
+
+The motivation of this CEP is merely informative. It describes the details of an existing grammar.
+
+## Nomenclature
+
+The `MatchSpec` query syntax is a mini-language designed to query package records from one or more conda channels. It is sometimes referred to as simply _spec_.
+
+## Mini language
+
+The `MatchSpec` mini language has gone through several iterations.
+
+The simplest form merely consists of up to three positional arguments: `name [version [build]]`. Only `name` is required. `version` can be any version specifier. `build` can be any string matcher. See "Match conventions" below.
```

Comment:
```suggestion
The simplest form merely consists of up to three positional arguments: `name [version [build]]`. Only `name` is required. `version` can be any [version specifier](#version-specifier). `build` can be any [string matcher](#string-matching). See [Match conventions](#match-conventions) below.
```

Also, should we define what characters are accepted in a package name?

Reply:
I feel this is going to be part of a different CEP, `PackageRecord`.